src/service/action-impl.js

```diff
@@ -46,6 +47,11 @@ const ELEMENTS_ACTIONS_MAP_ = {
   'AMP': ['setState'],
 };
 
+/** @const {!Object<string, !Array<string>>} */
+const WHITELISTED_INPUT_DATA_ = {
```

Comment:
I was just about to create an FR for something similar to this! Love it when what I need is already done :)

Well, I actually needed `checked` for `radio` and `checkbox` as well. Could we please those in this PR as well? Thanks!